app/src/main/java/in/ac/bits_pilani/goa/ard/activities/AboutMAC.java

```diff
@@ -0,0 +1,159 @@
+package in.ac.bits_pilani.goa.ard.activities;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.text.Html;
+import android.util.Log;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.load.engine.DiskCacheStrategy;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import in.ac.bits_pilani.goa.ard.R;
+import in.ac.bits_pilani.goa.ard.utils.AHC;
+
+/**
+ * AboutMac class.
+ * @author Aayush
+ * @version 1.0.
+ */
+
+public class AboutMAC extends AppCompatActivity {
+
+    /**
+     * About Mac Text.
+     */
+    @BindView(R.id.about_mac_text)
+    TextView textView;
+
+    /**
+     * About Mac Image.
+     */
+    @BindView(R.id.about_mac_image)
+    ImageView imageView;
+
+    /**
+     * About Mac Toolbar.
+     */
+    @BindView(R.id.toolbar_about_mac)
+    Toolbar toolbar;
+
+    /**
+     * Database Reference.
+     */
+    DatabaseReference databaseReference;
+
+    /**
+     * Vale event listener.
+     */
+    ValueEventListener listener;
+
+    /**
+     * Sharedpreferences.
+     */
+    SharedPreferences sharedPreferences;
+
+    /**
+     * String html.
+     */
+    String html;
+
+    /**
+     * String url.
+     */
+    String url;
+
+    /**
+     * String shtml.
+     */
+    String shtml = "html";
+
+    /**
+     * String surl.
+     */
+    String surl = "imageUrl";
+
+    /**
+     * Float magicnumber.
+     */
+    float magicnumber = 0.5f;
+
+    /**
+     * constructor.
+     */
+    public AboutMAC(){
```

Comment:
:app:checkstyle[ant:checkstyle] [ERROR] /home/travis/build/MobileApplicationsClub/ARD/app/src/main/java/in/ac/bits_pilani/goa/ard/activities/AboutMAC.java:32: Class should define a constructor. [MissingCtor]